webapp/graphql/Search.js

```diff
@@ -52,6 +61,20 @@ export const searchPosts = gql`
   }
 `
 
+export const searchGroups = gql`
+  ${groupFragment}
+
+  query ($query: String!, $firstGroups: Int, $groupsOffset: Int) {
+    searchGroups(query: $query, firstGroups: $firstGroups, groupsOffset: $groupsOffset) {
```

Comment:
Also naming things ehre - `firstGroups` just sounds wrong - not sure I fully understand this

Reply:
Plural is correct in my eyes.
See somewhere above …